services/callback/src/main/java/app/coronawarn/server/services/callback/config/SecurityConfig.java

```diff
@@ -28,8 +28,13 @@
 
   private static final String CALLBACK_ROUTE =
       "/version/v1" + CallbackController.CALLBACK_ROUTE;
-  private Logger logger = LoggerFactory.getLogger(SecurityConfig.class);
-  private CallbackServiceConfig callbackServiceConfig;
+  private static final String ACTUATOR_ROUTE = "/actuator/";
+  private static final String HEALTH_ROUTE = ACTUATOR_ROUTE + "health";
+  private static final String PROMETHEUS_ROUTE = ACTUATOR_ROUTE + "prometheus";
+  private static final String READINESS_ROUTE = ACTUATOR_ROUTE + "readiness";
```

Comment:
Can we verify that this URL is correct? I tried it locally and since readiness is custom group of health it should be under health/readiness?
```suggestion
  private static final String READINESS_ROUTE = HEALTH_ROUTE + "/readiness";
```